README.md

```diff
@@ -14,6 +14,12 @@ You need to have already installed ROS2 (tested on humble distro).
 pip install ros2-graph
 ```
 
+To be able to export the diagram as an image install [mermaid-cli](https://github.com/mermaid-js/mermaid-cli) on your computer
+
+```
+npm install -g @mermaid-js/mermaid-cli
+```
+
```

Comment:
I think the user should also install node and npm right?

Reply:
You are rigth